PayItForward.UnitTests/Web/Controllers/HomeController/MyStories_Should.cs

```diff
@@ -0,0 +1,120 @@
+﻿namespace PayItForward.UnitTests.Web.Controllers.HomeController
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Security.Claims;
+    using System.Text;
+    using AutoMapper;
+    using Microsoft.AspNetCore.Http;
+    using Microsoft.AspNetCore.Mvc;
+    using Moq;
+    using PayItForward.Models;
+    using PayItForward.Services.Abstraction;
+    using PayItForward.UnitTests.Web.Controllers.Stubs;
+    using PayItForward.Web.Models.HomeViewModels;
+    using PayItForward.Web.Models.StoryViewModels;
+    using Xunit;
+
+    public class MyStories_Should
+    {
+        private readonly PayItForward.Web.Controllers.HomeController homeController;
+        private readonly Mock<IStoriesService> storiesService;
+        private readonly Mock<ICategoriesService> categoriesService;
+        private readonly Mock<IMapper> mapper;
+        private readonly Mock<IHttpContextAccessor> httpaccessor;
+
+        public MyStories_Should()
+        {
+            this.mapper = new Mock<IMapper>();
+            this.storiesService = new Mock<IStoriesService>();
+            this.categoriesService = new Mock<ICategoriesService>();
+            this.httpaccessor = new Mock<IHttpContextAccessor>();
+            this.homeController = new PayItForward.Web.Controllers.HomeController(
+                this.storiesService.Object, this.categoriesService.Object, this.mapper.Object, this.httpaccessor.Object);
+        }
+
+        [Fact]
+        public void ReturnMyStoriesViewModel_WithAListOfBasicStoryViewModel()
+        {
+            // Arrange
+            var dtoStories = HomeController_Stubs.GetTestListWithStoryDtos();
```

Comment:
You can set one of the stories with different user so the test can check if two from three stories are returned based on the user